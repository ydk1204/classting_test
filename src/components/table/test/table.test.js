import React from "react";
import renderer from 'react-test-renderer';
import Table from '../table';

const tableHead = ['안녕', '하세요'];
const resultData = ['감사해요', '잘있어요'];

describe('Table', () => {
  it('renders', () => {
    const component = renderer.create(<Table tableHead={ tableHead} resultData={resultData} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});