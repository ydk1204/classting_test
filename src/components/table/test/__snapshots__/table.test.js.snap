// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table renders 1`] = `
<div
  className="flex justify-center w-full"
>
  <table>
    <thead>
      <tr>
        <th>
          안녕
        </th>
        <th>
          하세요
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          감사해요
        </td>
        <td>
          잘있어요
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
